src/crypto/crypto_ec.cc

```diff
@@ -839,10 +839,9 @@ Maybe<void> ExportJWKEcKey(
   return JustVoid();
 }
 
-Maybe<bool> ExportJWKEdKey(
-    Environment* env,
-    std::shared_ptr<KeyObjectData> key,
-    Local<Object> target) {
+Maybe<void> ExportJWKEdKey(Environment* env,
```

Comment:
Nit: Why don't we just return bool?

Reply:
because it would be less explicit

Reply:
It's V8 convention to return `Nothing` when an exception is pending and `Just<T>` when the operation succeeds with a result of type `T`. We do follow this convention in many places, and `T` just happens to be `void` here.